LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -625,10 +628,11 @@ private async Task<LoRaADRResult> PerformADR(LoRaRequest request, LoRaDevice loR
             return loRaADRResult;
         }
 
-        private static bool ValidateRequest(LoRaRequest request, bool isFrameCounterFromNewlyStartedDevice, ushort payloadFcnt, LoRaDevice loRaDevice, bool requiresConfirmation, out bool isConfirmedResubmit, out LoRaDeviceRequestProcessResult result)
+        private static bool ValidateRequest(LoRaRequest request, bool isFrameCounterFromNewlyStartedDevice, uint payloadFcnt, LoRaDevice loRaDevice, bool requiresConfirmation, out bool isConfirmedResubmit, out LoRaDeviceRequestProcessResult result)
         {
             isConfirmedResubmit = false;
             result = null;
+
```

Comment:
As previous, if we add original payload fcnt, we should add them here too (must pass additional parameter)